app/models/to_do_entry.rb

```diff
@@ -1,3 +1,46 @@
 class ToDoEntry < ApplicationRecord
+
+    def sort_date(other)
+
+        self_due_date = self[:due_date]
+        other_due_date = other.due_date
+        
+        if self_due_date == nil && other_due_date == nil
+            return -1
+        end 
+
+        if self_due_date == nil && other_due_date != nil
+            return 1
+        end
+
+        if self_due_date != nil && other_due_date == nil
+            return -1
+        end
+
+        if other_due_date == self_due_date 
+            0
+        elsif self_due_date > other_due_date
+            1
+        else 
+            -1
+        end
+    end 
+
+    def formatted_date
+        if self[:due_date]
+            self[:due_date].to_formatted_s(:long)     
+        end
+    end 
+
+    def due_date_passed? 
```

Comment:
I understand this is used in your view to potentially add a class name to an entry, but the convention in Ruby for methods ending with a `?` is for them to return a boolean.

Would you be able to change this method to return true or false, and then add the class name `"past-due"` to the to do entry if it's true?